docs/usage.md

```diff
@@ -348,83 +349,136 @@ This will affect the add button text and index/form page titles.
+fae.pages_path
+```
+
+## Top Nav
+
+Fae has an alternative top navigation that's ideal if you have a lot of objects or want to nest them more than two levels.
+
+![Top Nav](images/navigation.png)
+
+To enable the top nav, add this option to `config/initializers/fae.rb`:
+
+```ruby
+config.has_top_nav = true
+```
+
+### Top/Side Nav Hierarchy
 
-When sublinks are present, the main nav item will trigger a drawer holding the sublinks to open/close. Add sublinks using the following format:
+When the top nav is enabled, the first and second level of `structure` defines the main nav and dropdown items respectively. When you click into a section that has third and/or fourth levels, a side nav will appear to display those items.
 
 ```ruby
-def nav_items
+def structure
   [
-    {
-      text: 'Items with sublinks', sublinks: [
-        { text: 'Item Sublink 1', path: main_app.admin_some_path },
-        { text: 'Item Sublink 2', path: main_app.admin_someother_path }
-      ]
-    }
+    item('Top nav item', subitems: [
+      item('Top nav dropdown item', subitems: [
+        item('Side nav item', subitems: [
+          item('Side nav nested item', path: '#')
+        ])
+      ]),
+    ])
   ]
 end
 ```
 
-## Dynamic Content in Nav
+If the main nav dropdown doesn't have a defined `path`, it will search the `subitems` for the first available path to link to.
 
-Dynamic content is allowed in the the `nav_items` concern. Here's an example:
+Note: the side nav is hidden in forms by design, this removes clutter and allows the user to focus on the content of that object.
+
+## Dynamic Content in the Navigation
+
+Dynamic content can be defined in `navigation_concern.rb`. Here's an example:
 
 ```ruby
 module Fae
-  module NavItems
+  module NavigationConcern
     extend ActiveSupport::Concern
 
-    def nav_items
+    def structure
       [
-        { text: 'Releases', path: main_app.admin_releases_path },
-        { text: 'Tiers', sublinks: tier_sublinks }
+        item('Teams', subitems: team_subitems)
+        # scaffold inject marker
       ]
     end
 
     private
 
-    def tier_sublinks
-      tiers_arr = [{ text: 'New Tier', path: main_app.new_admin_tier_path }]
-      Tier.each do |tier|
-        tiers_arr << { text: tier.name, path: main_app.edit_admin_tier_path(tier) }
+    def team_subitems
+      teams_arr = [ item('Team List', path: admin_teams_path) ]
+      Team.all.each do |team|
+        teams_arr << item(team.name, subitems: [
+          item('Coaches', path: admin_team_coaches_path(team)),
+          item('Players', path: admin_team_players_path(team))
+        ])
       end
-      tiers_arr
+      teams_arr
     end
-
   end
 end
 ```
 
 ---
 
+# Global Search
+
+Fae features a global search/navigation in the upper right. Clicking on this will automatically display the first two levels of navigation. By entering at least three characters in the search bar results will be displayed from both navigation items and objects in your DB.
```

Comment:
> By entering at least three characters in the search bar results will be displayed from both navigation items and objects in your DB.

Maybe "Search results will be displayed from both navigation items and DB objects when three characters or more are entered in the search bar" ?